tests/integration/server/courseInstance/courseInstance.controller.test.ts

```diff
@@ -53,6 +53,7 @@ describe('CourseInstance API', function () {
   let ciRepository: Repository<CourseInstance>;
   let api: HttpServer;
   let authStub: SinonStub;
+  let configService: ConfigService;
 
```

Comment:
I'd like to see the constants used at the top , so I like to see `let currentAcademicYear = 2022` here. But this is not important and its up to you.

Reply:
Also if you can put a simple comment about `currentAcademicYear`

Reply:
Sure thing, the comment was added for `currentAcademicYear` in [3e4fe28](https://github.com/seas-computing/course-planner/pull/531/commits/3e4fe28077f0c87b76831af794fabab71be52a98) and it was moved to the top after `configService`.